Assets/Prefabs/Items/Letter.prefab

```diff
@@ -69,7 +69,7 @@ Transform:
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 178772}
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: 0, y: .5, z: 0}
+  m_LocalPosition: {x: 10.0500002, y: -.349999994, z: 0}
```

Comment:
Probeer in de prefabs niet de locatie aan te passen, als je prefabs in je scene sleept weet je dan zeker dat het op x0y0z0 komt te staan.